src/App.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable*/
```

Comment:
- [ ] Please, consider removing the linter disables from all the code of your app - let linter makes its direct job: find bugs on early steps of the development 👍